index.ftd

```diff
@@ -152,148 +192,125 @@ Body text is here in h3
 
 
 
--- ftd.column h0:
+-- component h2:
 caption title:
 optional body body:
-width: fill
-region: h0
-append-at: inner
-margin-top: 0
-margin-bottom: 32
 
+-- ftd.column:
+;;append-at: inner -- TODO FTD 0.3
+width: fill-container
+region: h2
+margin-top.px: 32
+margin-bottom.px: 16
 
---- ftd.text:
-caption $title:
-region: title
-role: $fpm.type.heading-hero
-text: $title
-color: $fpm.color.main.text-strong
-padding-bottom: 16
-padding-top: 8
-$on-click$ if $open: toggle $open
+-- ftd.row:
+width: fill-container
+spacing.px: 20
 
---- ftd.column:
-id: inner
-width: fill
-margin-bottom if $body is not null: -26
+-- ftd.text: $h2.title
+role: $inherited.types.heading-large
+color: $inherited.colors.text-strong
+padding-bottom.px: 16
+padding-top.px: 8
 
---- markdown:
-if: $body is not null
-body: $body
+-- end: ftd.row
 
+-- ftd.column:
+;;id: inner -- TODO FTD 0.3
+width: fill-container
+margin-bottom.px if {$h2.body != NULL}: -26
```

Comment:
Don't use `$` in FScript mode (i.e. inside `{}`). Also, use space after `{` and before `}`.
```suggestion
margin-bottom.px if { h2.body != NULL }: -26
```